meinberlin/react/contrib/ToggleSwitch.jsx

```diff
@@ -3,27 +3,26 @@ import React from 'react'
 export const ToggleSwitch = (props) => {
   const {
     onSwitchStr,
-    offSwitchStr,
     uniqueId,
-    toggleSwitch
+    toggleSwitch,
+    defaultChecked
   } = props
 
   return (
-    <div className="switch__group">
-      <label htmlFor={uniqueId} className="switch__label">
-        <span className="switch__label-text" aria-hidden="true">{onSwitchStr}</span>
-        <input
-          className="switch__input"
-          type="checkbox"
-          data-check-switch=""
-          role="switch"
-          id={uniqueId}
-          name={uniqueId}
-          onChange={toggleSwitch}
-          aria-label={offSwitchStr}
-        />
-        <span className="switch__toggle" aria-hidden="true" />
-      </label>
+    <div className="toggle-switch form-check">
+      <label className="toggle-switch__label" htmlFor={uniqueId}>{onSwitchStr}</label>
+      <input
+        type="checkbox"
+        name={uniqueId}
+        id={uniqueId}
+        className="toggle-switch__input"
+        onChange={toggleSwitch}
+        defaultChecked={defaultChecked}
+      />
+      <span className="toggle-switch__display" hidden>
```

Comment:
I was thinking the same, but I would rather not just do it for the sake of it. In react there are controlled and uncontrolled components. But I wouldn't handle the state within that component and only offer a callback like currently `toggleSwitch`. Then this component would take a value and if it has both a value and a onChange handler, it becomes a controlled component, otherwise it will be an uncontrolled one. Both should be fine though. I can adapt the component in a way that it also supports being controlled though.